mPower2/mPower2/Info.plist

```diff
@@ -20,12 +20,16 @@
 	<string>2.1</string>
 	<key>CFBundleVersion</key>
 	<string>29</string>
+	<key>LSApplicationCategoryType</key>
+	<string></string>
```

Comment:
?

If included, shouldn't this have a category set?

Reply:
Probably. I have no idea how this even got in here; I didn't add it, at least not intentionally. Removing.